config/locales/en.yml

```
@@ -599,6 +599,10 @@ en:
           <p class="govuk-body">You'll need the email or username and password you used to create an account with %{idp_name}.</p>
         continue_verifying: 'Continue verifying with %{idp_name}'
         return_html: 'Or return to <a href="%{rp_start_page}">%{rp_name}</a>.'
+      idp_no_longer_providing_registrations:
+        title: "Verification Paused %{idp_name}"
+        heading: "%{idp_name} are no longer providing registrations"
```

```suggestion
        heading: "You can no longer create an identity account with %{idp_name}"
```
This is a variation of the text on the `/about` page
> You create an identity account with a company that meets government privacy and security standards.

This should be reviewed by @sabrinaharris for final approval

---

Yep, Sabrina is going to take the content to the content folks.